src/elements/content-sidebar/activity-feed/task-new/Task.js

```diff
@@ -319,7 +335,7 @@ class Task extends React.Component<Props, State> {
                         </div>
                         <div className="bcs-Task-status">
                             <TaskStatus status={status} />
-
+                            <TaskMultiFileIcon isMultiFile={isMultiFile} />
```

Comment:
Seems interesting to pass in a prop which determines whether the Icon will render or not. What are your thoughts on letting the Icon always render and move the condition to the client logic, so `{ isMultiFile && <TaskMultiFileIcon /> }`

Reply:
https://github.com/box/box-ui-elements/pull/2049#discussion_r398274700

Reply:
Gotcha, thanks for the context 👍